src/View.php

```diff
@@ -749,37 +770,22 @@ public function render($force_echo = true)
      */
     public function renderJSON($force_echo = true, $region = null)
     {
-        try {
-            $this->renderAll();
-
-            return json_encode(['success' => true,
-                                'message' => 'Success',
-                                'atkjs'   => $this->getJS($force_echo),
-                                'html'    => $this->template->render($region),
-                                'id'      => $this->name, ]);
-        } catch (\Exception $exception) {
-            $this->_rendered = false;
-            $l = $this->add(new self());
-            if ($exception instanceof \atk4\core\Exception) {
-                $l->template->setHTML('Content', $exception->getHTML());
-            } elseif ($exception instanceof \Error) {
-                $l->add(new self(['ui' => 'message', get_class($exception).': '.
-                                                            $exception->getMessage().' (in '.$exception->getFile().':'.$exception->getLine().')',
-                    'error', ]));
-                $l->add(new Text())->set(nl2br($exception->getTraceAsString()));
-            } else {
-                $l->add(new self(['ui' => 'message', get_class($exception).': '.$exception->getMessage(), 'error']));
-            }
+        $this->renderAll();
```

Comment:
raise exception to upper level
we already have exception handler that format correctly the exception, if an exception happen here will broke the normal flow of the whole application.